### ISSUE TITLE
Require confirmation before starting visualization tools for large lists

### ISSUE THREAD

**Issue by heralden** [intermine/bluegenes#630]
Some tools on the results page grows linearly (or worse) with the amount of elements.

Right now the burden is on the tool developer to handle this (usually with a confirmation dialog when there are above a set amount of elements). It might be better to have Bluegenes handle this instead.

One approach could be having list visualization tools optionally specify a `threshold` key in their *config.json*, which when surpassed will make Bluegenes block the visualization from running and require the user to click a confirmation button. This could simply be implemented by leaving the tool header collapsed, and adding text explaining why and that the user needs to click it.

A related but still separate issue: #582

*Update: In addition to this, we should also set hard limits (that are still overridable with a button press). For loading the tools section, this should be 1000 items. For enrichment, 2000 items.*
